app/screens/post_options/index.ts

```diff
@@ -57,7 +57,7 @@ const observeCanEditPost = (database: Database, isOwner: boolean, post: PostMode
 const withPost = withObservables([], ({post, database}: {post: Post | PostModel} & WithDatabaseArgs) => {
     let id: string | undefined;
     let combinedPost: Observable<Post | PostModel | undefined> = of$(undefined);
-    if (post.type === Post.POST_TYPES.COMBINED_USER_ACTIVITY && post.props?.system_post_ids) {
+    if (post?.type === Post.POST_TYPES.COMBINED_USER_ACTIVITY && post.props?.system_post_ids) {
```

Comment:
Not sure why, but the app kept crashing due to `post` sometimes being undefined on app reload

Reply:
This is weird. What do you mean exactly by reload?

Reply:
@larkox This is during development, and I believe it can occur when the app is refreshed from the JS server. I will try and pinpoint this issue